lib/main.js

```diff
@@ -60,9 +60,9 @@ export default {
       requireResolve(TSLINT_MODULE_NAME, { basedir },
         (err, linterPath, pkg) => {
           let linter;
-          if (!err && pkg && pkg.version.startsWith('3.')) {
+          if (!err && pkg && pkg.version.startsWith('4.')) {
```

Comment:
Since this project can support loading `tslint` from a Project's folder, we should probably support as many different versions of `tslint` as we can, assuming it's not that much work. Since it seems that v3 -> v4 isn't that big of a change I'd prefer if _both_ versions were supported, so just an `else if` here for v4.